src/frontend/src/catalog/system_catalog/pg_catalog/pg_matviews.rs

```diff
@@ -24,10 +24,10 @@ use risingwave_frontend_macro::system_catalog;
     "SELECT schemaname,
             i.relationname AS matviewname,
             i.relationowner AS matviewowner,
-            definition,
-            i.relationid AS matviewid,
-            i.relationtimezone AS matviewtimezone,
-            i.fragments AS matviewgraph
+            NULL AS tablespace,
+            false AS hasindexes,
+            true AS ispopulated,
+            definition
         FROM rw_catalog.rw_relation_info i
```

Comment:
I think we should change this view to use `rw_materialized_views` and `rw_schemas` instead. Query on `rw_relation_info` still issues a reader on `fragment`. 
Btw I will open another new PR to move these extra streaming info in `rw_relation_info` to a new catalog. It's not very reasonable to be placed them in `rw_relation_info`.
https://github.com/risingwavelabs/risingwave/blob/34d31aa68b604fc4744e75b280075e45146df9d8/src/frontend/src/catalog/system_catalog/rw_catalog/rw_relation_info.rs#L22-L24

Reply:
> I think we should change this view to use `rw_materialized_views` and `rw_schemas` instead

Let me try.